src/types/stream/stream.ts

```diff
@@ -15,9 +15,19 @@ export enum StreamingState {
 export enum ConnectivityState {
     Strong = 'STRONG',
     Weak = 'WEAK',
-    Unknown = 'UNKNOWN'
+    Unknown = 'UNKNOWN',
 }
 
+export enum AgentState {
```

Comment:
This is a very confusing name. It can refer to a lot of stuff.
Think of something better.